docs/Validation-and-Serialization.md

```diff
@@ -302,6 +303,20 @@ fastify.setErrorHandler(function (error, request, reply) {
 })
 ```
 
+### JSON Schema and Shared Schema support
+
+JSON Schema has some type of utilities in order to optimize your schemas that,
+in conjuction with the Fastify's shared schema, let you reuse all your schemas easily.
+
+| Use Case                          | Validator | Serializer |
+|-----------------------------------|-----------|------------|
+| shared schema                     | ✔️ | ✔️ |
+| `$ref` to `$id`                   | ✔ | ✔️ |
+| `$ref` to `/definitions`          | ✔️ | ✔️ |
+| `$ref` to shared schema `$id`          | ❌ | ✔️ |
+| `$ref` to shared schema `/definitions` | ❌ | ✔️ |
```

Comment:
I think it would be good to add some examples

Reply:
Done 👍